modules/auxiliary/dos/ssl/dtls_changecipherspec.rb

```diff
@@ -6,7 +6,7 @@
 class MetasploitModule < Msf::Auxiliary
   include Msf::Auxiliary::Dos
   include Msf::Exploit::Capture
-  include Exploit::Remote::Tcp
+  include Exploit::Remote::Udp
```

Comment:
How about we register `RHOST(S?)` and `RPORT`? It looks like this mixin is included only for the options.

Reply:
OK, that sounds reasonable, I'll fix that.